start_pg_example.sh

```diff
@@ -0,0 +1,20 @@
+DB_USER="postgres"
+DB_PASSWORD="password"
+DB_NAME="test"
+DB_HOSTNAME="localhost"
+DB_PORT=5432
+DB_DIALECT="postgres"
```

Comment:
These env vars should be in the .env.defaults and read unless overwritten in a private .env file